cmd/tsr/api.go

```diff
@@ -18,7 +22,17 @@ type apiCmd struct {
 }
 
 func (c *apiCmd) Run(context *cmd.Context, client *cmd.Client) error {
-	err := config.Check([]config.Checker{CheckProvisioner, CheckBeanstalkd, CheckBasicConfig})
+	log.Printf("Opening config file: %s\n", configPath)
+	err := config.ReadConfigFile(configPath)
+	if err != nil {
+		msg := `Could not open tsuru config file at %s (%s).
+  For an example, see: tsuru/etc/tsuru.conf
+  Note that you can specify a different config file with the --config option -- e.g.: --config=./etc/tsuru.conf`
+		log.Fatalf(msg, configPath, err)
+		os.Exit(1)
+	}
+	fmt.Printf("Done reading config file: %s\n", configPath)
```

Comment:
You should use `fmt.Fprintf` instead `fmt.Printf`:

    fmt.Fprintf(context.Stdout, "Done reading config file: %s\n", configPath)